geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/ConverterIngest.scala

```diff
@@ -77,7 +79,12 @@ class ConverterIngest(sft: SimpleFeatureType,
     new LocalIngestConverterImpl(sft, path, converters, failures)
 
   override def runDistributedJob(statusCallback: StatusCallback): (Long, Long) = {
-    new ConverterIngestJob(dsParams, sft, converterConfig, inputs, libjarsFile, libjarsPaths).run(statusCallback)
+    // Check conf if we should run against small files and use Combine* classes accordingly
+    if (mode.contains(RunModes.DistributedCombine)) {
```

Comment:
I think we need to add this mode to the docs, right?

Reply:
Added